### ISSUE TITLE
Second-based date precision (date_seconds) for wider range

### ISSUE THREAD

**Issue by mitar** [elastic/elasticsearch#83101]
TLDR: I am asking for `date_seconds` field type.

I am building an index of various events from human and universe history. I am using Wikidata to populate events and you have for example [Agelacrinitidae](https://www.wikidata.org/wiki/Q35144012) which were present on Earth from 449500000 years BCE to 272500000 years BCE. Currently, with `date` field type it is not possible to represent this as it is outside of the `long` range because timestamps use millisecond precision.

On the other hand, if timestamp would have (only) second precision (which is very precise when looking at events at this level) then it would be possible to represent any timepoint since the beginning of this universe and after its end.

So I am asking for `date_seconds` field type. There is already a precedence with `date_nanos`, but `date_seconds` would go into the other direction and make the range wider at the expense of precision. Which is fine for probably all use cases where you want such a wide range.

**Comment by nik9000** [comment]
I filed a similar but different issue a while back #9048.

**Comment by mitar** [comment]
So any update on this one?
